front/app/modules/commercial/content_builder/admin/components/ContentBuilderEditModePreview/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { memo, useState } from 'react';
+
+// components
+import { Box, Icon, Button } from '@citizenlab/cl2-component-library';
+
+// styles
+import { stylingConsts } from 'utils/styleUtils';
+
+// hooks
+import useLocale from 'hooks/useLocale';
+
+type ContentBuilderMobileViewProps = {
+  projectId: string;
+};
+
+const ContentBuilderEditModePreview = React.forwardRef<
+  HTMLIFrameElement,
+  ContentBuilderMobileViewProps
+>(({ projectId }, ref) => {
+  const locale = useLocale();
+  const [isMobile, setIsMobile] = useState(true);
+  const whiteIfMobile = isMobile ? '#FFFFFF' : '#044D6C';
+  const whiteIfDesktop = isMobile ? '#044D6C' : '#FFFFFF';
```

Comment:
I think the #044D6C colour needs to come from the named colours. You can find the colour names [here](https://citizenlabdotco.github.io/cl2-component-library/?path=/story/design-colors--default-story), or check with Ben  what he used? The import would be: `import { colors } from 'utils/styleUtils';`

The variable names are also maybe a bit confusing for me. What do you think about using something like "colorIfMobileView/colorIfDesktopView" instead?